defaults/main.yml

```diff
@@ -3,3 +3,6 @@ vertica_catalog_dir: /var/vertica/catalog
 vertica_data_dir: /var/vertica/data
 vertica_ips: 127.0.0.1
 skip_install: false
+run_mode: Deploy
+vertica_env: {LC_ALL: en_US.UTF-8, LANG: en_US.UTF-8}
+vertica_database_name: mon
```

Comment:
It says this file doesn't have a newline at the end. You should add that as lack of it confuses some tools